### ISSUE TITLE
Question of topk_proposals

### ISSUE THREAD

**Issue by Christinepan881** [fundamentalvision/Deformable-DETR#134]
In `deformable_transformer.py`, the code to select the topk proposals:
`topk_proposals = torch.topk(enc_outputs_class[..., 0], topk, dim=1)[1]`.
Why use score for class index 0 as the sorting reference?
It seems that the `num_classes` in `enc_outputs_class` is the same as the `num_classes` of the whole dataset.

**Comment by volgachen** [comment]
I am also confused about this configuration.

It seems `enc_outputs_class` is _91-d_ in the last dimension, and _0-index_ does not correspond to any category.
Is there any supervision that works on the _0-index_ logits?
